rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -92,56 +101,65 @@ Time zone for defining downtimes.
 Click *Create* to add a downtime to the application.
 . Click *Save*.
 
-Learn how to xref:myrpa-start.adoc#required-process-applications[assign required applications to a process].
-
-Learn how to xref:botmanagement-manage.adoc#bot-assign-servicetimes-applications[assign required applications to a bot].
-
 ==== Edit an Application
 
 Edit an application to change its data or down-times:
 
-. In the *Process Applications* view of the *Process Automation* module, click *Create*.
+. Open the *Process Applications* view of the *Process Automation* module.
+. Click *Create*.
 . Edit the form *Edit the application*.
++
+The properties are explained in the form <<form-create-process-applictions, *Create a new application*>>.
 . Click *Save*.
 
 ==== Deactivate or Activate an Application
 
 Deactivate an application if it is temporarily not available and you don't know when it will be available again.
 
-Use down-times to schedule maintenance and down times. Deactivation periods are not added to the list of down-times.
+Use down-times to schedule maintenance and planned down times. Deactivation periods are not added to the list of down-times.
+
+RPA Bots execute processes whose applications are currently down to completion and do not restart them until the downtime or deactivation period has ended.
```

Comment:
The word Bots is lowercase elsewhere.

Reply:
(sentence replaced) Here I refer to the product RPA Bot. We have four RPA products: RPA Manager, RPA Recorder, RPA Builder and RPA Bot. In the documentation in most cases I just use 'bot' instead of 'RPA Bot'. But it is the only type of bot working with the other RPA products at the moment. Should I replace 'bot' with 'RPA Bot' everywhere?

Reply:
@dhirsch1